website/docs/helpers.mdx

```diff
@@ -4,55 +4,67 @@ title: Helpers
 sidebar_label: Helpers
 ---
 
-### `createNavigationCommands`
+### `setRoot`
 
-A set of overloaded methods which reduce the boilerplate required to execute common navigation commands.
+Overloaded method which reduce the boilerplate required for the `setRoot` command.
```

Comment:
should say "Overloaded method which reduces..."

Reply:
thanks! fixed in 93c6fc57642804cf6f30db5841a139349273b188